data/remnant/remnant 1 introduction.txt

```diff
@@ -2164,12 +2121,13 @@ mission "Remnant: Return the Samples 2"
 
 
 # Compatibility patch for those who completed Return the Samples 2 before 0.9.11, where it did not have the timer.
-mission "Remnant Return the Samples Compatibility Patch"
+mission "Remnant: Return the Samples Compatibility Patch"
 	landing
 	invisible
 	to offer
 		has "Remnant: Return the Samples 2: done"
 		not "event: remnant: return the samples timer"
+		not "Remnant Return the Samples Compatibility Patch: offered"
```

Comment:
This one probably doesn't need to block its old version at all since the event is only used as a delay trigger.
```suggestion
```